node/full.go

```diff
@@ -384,10 +396,11 @@ func (n *FullNode) Run(ctx context.Context) error {
 		return fmt.Errorf("error while starting data sync service: %w", err)
 	}
 
+	go n.reaper.Start()
+
 	if n.nodeConfig.Node.Aggregator {
 		n.Logger.Info("working in aggregator mode", "block time", n.nodeConfig.Node.BlockTime)
-
-		go n.blockManager.BatchRetrieveLoop(ctx)
+		// go n.blockManager.BatchRetrieveLoop(ctx)
```

Comment:
BatchRetrieveLoop is not needed for based sequencing. Also, not needed for single sequencing Proposer nodes. it is only needed for full nodes in the single seq.